### PR TITLE
feature/filter-rpc-script

### PR DESCRIPTION
**Pull request by decanus**

### CODE REVIEW

File: docs/tutorial/filter.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# Run another node with staticnode argument
+./build/wakunode2 --ports-shift:1 --staticnode:/ip4/0.0.0.0/tcp/60000/p2p/16Uiu2HAmF4tuht6fmna6uDqoSMgFqhUrdaVR6VQRyGr6sCpfS2jp --filternode:/ip4/0.0.0.0/tcp/60000/p2p/16Uiu2HAmF4tuht6fmna6uDqoSMgFqhUrdaVR6VQRyGr6sCpfS2jp
+```
+
+You should see your nodes connecting.
+
+Do basic RPC calls:
+
+```
+./build/rpc_subscribe 8545
```
Comment (oskarth) [inline docs/tutorial/filter.md:28]: I don't understand how you specify that you want to be a "light node" and why we are subscribing here

Comment (oskarth) [reply docs/tutorial/filter.md:28]: Even if we don't do this, is it 100% the node isn't subscribing by default?

**Comment by oskarth** [comment]
Is this tested?

**Comment by decanus** [comment]
@oskarth see the screenshot. Proof of tested.

**Comment by oskarth** [comment]
Is the log message different for subscribe and subscribe content? I think we should remove the subscribe normal part.

**Comment by decanus** [comment]
Yeah, the messages are different. Ok @oskarth can remove.

**Review by oskarth — APPROVED** [review]

**Pull request merged**